pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/ASTConditionalExpression.java

```diff
@@ -32,13 +31,23 @@ public ASTConditionalExpression(JavaParser p, int id) {
         super(p, id);
     }
 
+
+    /**
+     * @deprecated To be removed in 7.0.0
+     */
+    @Deprecated
     public void setTernary() {
-        isTernary = true;
     }
```

Comment:
you will need to add a comment (a `// noop` at least) for PMD to pass on itself

Reply:
Sure thing